app/src/pages/inside/projectSettingsPageContainer/content/notifications/notifications.jsx

```diff
@@ -167,10 +186,17 @@ export const Notifications = ({ setHeaderTitleNode }) => {
   useEffect(() => {
     if (notifications.length > 0) {
       setHeaderTitleNode(
-        <span className={cx('button')} onClick={onAdd}>
-          <Button disabled={!isAbleToEdit()} dataAutomationId={'createNotificationRuleButton'}>
-            {formatMessage(messages.create)}
-          </Button>
+        <span className={cx('button')}>
+          {isAbleToEdit() ? (
+            <CreateButton name={formatMessage(messages.create)} onClick={onAdd} />
```

Comment:
From my point of view, this component is redundant. Actually, he does nothing.
That is why I suggest is to remove it, and use the Button component directly.